### PR TITLE
Add narratable status for button actions in dynamic chart examples

### PR DESCRIPTION
**Pull request by krkshitij**
Two solutions:
1. `alert` role: This role makes screen readers announce an element immediately after its addition to the DOM (using JavaScript). While this works in most browsers and screen readers, it must be used with extreme caution as it interrupts the screen reader's current output. It is used to communicate an important and usually time-sensitive message to the user. Alerts are assertive live regions. Setting `role="alert"` is equivalent to setting `aria-live="assertive"` and `aria-atomic="true"`. For reference: [ARIA: alert role - Accessibility | MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Alert_Role)

2. `aria-live` attribute (_implemented_): This attribute marks specific areas of the HTML as live regions, informing screen readers to announce dynamic content changes. The `polite` value indicates that these changes should be announced at the next convenient time without interrupting the current announcement. Setting `aria-atomic="true"` signals to screen readers that they should announce the entire content rather than announcing only the changes. For reference: [How to announce new content after 'Show more' button is clicked? - Stack Overflow](https://stackoverflow.com/questions/69737344/how-to-announce-new-content-after-show-more-button-is-clicked)

Note:
> But in contrast to `role="alert"`, these live regions are not supported in a homogenous way by browsers and screen readers, so in our experience, they are very tricky to implement.

For reference: [Noticing screen readers using alert role - ADG](https://www.accessibility-developer-guide.com/examples/sensible-aria-usage/alert/)

## Previous Behavior

Screen reader doesn't announce any information when the 'Change data' or 'Change colors' button is clicked.

## New Behavior

Screen reader announces the status when the 'Change data' or 'Change colors' button is clicked.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/31dd521cf40a23d78d1169cbe4f31a107c782178'>31dd521cf40a23d78d1169cbe4f31a107c782178</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react-examples/src/react-charting/DonutChart/DonutChart.Dynamic.Example.tsx [hunk 1]
```diff
@@ -15,8 +15,21 @@ export interface IExampleState {
   hideLabels: boolean;
   showLabelsInPercent: boolean;
   innerRadius: number;
+  statusKey: number;
+  statusMessage: string;
 }
 
+const screenReaderOnlyStyle: React.CSSProperties = {
+  position: 'absolute',
+  width: '1px',
```
Comment (AtishayMsft) [inline packages/react-examples/src/react-charting/DonutChart/DonutChart.Dynamic.Example.tsx:24]: Could you add a description in the PR about this fix.
Why this css style.
What does aria-live and aria-atomic mean

Comment (AtishayMsft) [reply packages/react-examples/src/react-charting/DonutChart/DonutChart.Dynamic.Example.tsx:25]: also the alternative aria-alert option that you explored.

Comment (krkshitij) [reply packages/react-examples/src/react-charting/DonutChart/DonutChart.Dynamic.Example.tsx:25]: updated the PR description and added some comments

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**